doc/lua_api.txt

```diff
@@ -4893,6 +4893,12 @@ Utilities
 
 * `minetest.get_worldpath()`: returns e.g. `"/home/user/.minetest/world"`
     * Useful for storing custom data
+* `minetest.get_mod_data_path()`: returns e.g. ``"/home/user/.minetest/mod_data/mymod"`
+    * Useful for storing custom data *independently of worlds*.
+    * Must be called during mod load time.
+    * Can read or write to this directory at any time.
```

Comment:
Please note: Mods can write to each other's directories. I don't think this is a big problem as mods can already all write to the world dir. This should maybe be documented though.

This is to allow mods to be able to write after load time

Reply:
I think it's fine. Mods can mess with each other anyways, since they run in the same Lua environment.